src/main.rs

```diff
@@ -109,6 +109,11 @@ struct Opt {
     /// Keep cluster alive after the tool finishes
     #[structopt(long)]
     keep_cluster_alive: bool,
+
+    /// List of revisions to redirect to a target branch name which is used to generate a diff
+    /// while keeping unselected revision as is.
+    #[structopt(long, env)]
+    target_revision_redirect_selector: Option<String>,
```

Comment:
```suggestion
    /// List of target revisions to redirect to the target branch name while ignoring the rest. Default: Redirect all Applications to the target branch.
    #[structopt(long, env)]
    redirect_target_revisions: Option<String>,
```

Could we perhaps name it `redirect_target_revisions` instead? I feel that the term "selector" might imply a key-value relationship, as it is commonly associated with label or annotation selectors, which always involve key-value pairs.